app/javascript/components/Admin/Users/Comments.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import AdminCommentableComments from "$app/components/Admin/Commentable";
+import type { User } from "$app/components/Admin/Users/User";
+
+type AdminUserCommentsProps = {
+  user: User;
+};
+
+const AdminUserComments = ({ user }: AdminUserCommentsProps) => (
+  <AdminCommentableComments
+    count={user.comment_count}
```

Comment:
I think it's important that staff can see whether a user has comments before opening the dropdown - it would be inefficient having to click into it every time to check

Reply:
We could [cache the comments count](https://guides.rubyonrails.org/association_basics.html#counter-cache) to improve performance in a followup PR, because this will require an additional query upfront significantly increase the load time for user lists in the admin.

Reply:
Counter cache is defined as an option of a has_many relationship, so we can do it on a per-model/table basis. But yeah, paginating should be our first move.